csrank/choicefunction/ranknet_choice.py

```diff
@@ -0,0 +1,55 @@
+import functools
+
+import torch.nn as nn
+
+from csrank.choicefunction.choice_functions import SkorchChoiceFunction
+from csrank.modules.object_mapping import DenseNeuralNetwork
+from csrank.modules.scoring.ranknet import RankNetScoring
+
+
+class RankNetChoiceFunction(SkorchChoiceFunction):
+    """A variable choice estimator based on the RankNet-Approach.
+
+    See the documentation of the ``RankNetScoring`` class for more details.
+
+    Parameters
+    ----------
+    n_hidden : int
+        The number of hidden layers that should be used in the utility module.
+
+    n_units : int
+        The number of units per hidden layer that should be used in the utility
+        module.
+
+    activation : torch activation function (class)
+        The activation function that should be used for each layer of the
+        comparative network.
+
+    criterion : torch criterion (class)
+        The criterion that is used to evaluate and optimize the module.
```

Comment:
Very minor remark: The order of the parameters in the docstring and the __init__ signature is swapped.

Reply:
Good catch. Details matter. Changed :)